.cirrus.yml

```diff
@@ -27,7 +27,7 @@ test_task:
     - poetry config virtualenvs.in-project true
   test_script:
     - poetry install
-    - poetry run pytest -q --junitxml=junit.xml tests
+    - poetry run pytest -n auto -q --junitxml=junit.xml tests
```

Comment:
I think this change landed in the wrong commit

Reply:
Let's squash it.